tools/bumbershoot/idpassemble.xml

```diff
@@ -11,6 +11,7 @@
     </stdio>
     <command>
 <![CDATA[
+        export TMPDIR=\${TMPDIR:-.};
```

Comment:
@bernt-matthias I recall that TMPDIR is automatically set if we use some newer profile version. Is that correct? If so should we not use a higher profile?

Reply:
I also thought so, since there is also tmp and tmp.SOMEID directories within the job working dir. But I found only closed issues / PR in the Galaxy repo. Lets ask at gitter.

Reply:
Also profile 20.09 does not work